packages/devtools_app/test/support/mocks.dart

```diff
@@ -85,6 +85,9 @@ class FakeVmService extends Fake implements VmServiceWrapper {
     this._timelineData,
   );
 
+  /// Specifies the return value of `getHttpEnableTimelineLogging`.
+  bool getHttpEnableTimelineLoggingResult = true;
```

Comment:
nit: avoid starting member names with the word `get`

Reply:
+1. Perhaps `httpTimelineLoggingEnabled`

Reply:
This was to match the name of the method returning this value. Should that be changed as well?

Reply:
yes, get should also be removed from the method

Reply:
Done.